features/step_definitions/volume_snapshot.rb

```diff
@@ -7,3 +7,47 @@
     })
   step %Q/I switch to the default user/
 end
+
+# This step is to create VolumeSnapshotClass according to different paramater on differnt platforms, also it supports custom-defined parameter
+When /^admin creates a VolumeSnapshotClass replacing paths:$/ do |table|
+  path = "#{BushSlicer::HOME}/testdata/storage/snapshot"
+
+  aws_file = "#{path}/volumesnapshotclass-aws.yaml"
+  gce_file = "#{path}/volumesnapshotclass-gce.yaml"
+  azure_file = "#{path}/volumesnapshotclass-azure.yaml"
+  cinder_file = "#{path}/volumesnapshotclass-cinder.yaml"
+
+  iaas_type = env.iaas[:type] rescue nil
+  case iaas_type
+  when "aws"
+    file = aws_file
```

Comment:
I find the file names are very self-explanatory, so maybe remove the four variables and instead use:
`file = "#{path}/volumesnapshotclass-aws.yaml"` respectively.

Reply:
It makes sense, updated.